clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslClientAuthenticator.java

```diff
@@ -567,12 +577,12 @@ private boolean flushNetOutBuffer() throws IOException {
     private AbstractResponse receiveKafkaResponse() throws IOException {
         if (netInBuffer == null)
             netInBuffer = new NetworkReceive(node);
-        NetworkReceive receive = netInBuffer;
+        final NetworkReceive receive = netInBuffer;
```

Comment:
Some of these changes seem unrelated.

Reply:
That's a fair comment. I will revert these changes to keep the PR clean.